sdk/src/main/scala/io/horizen/account/utils/AccountFeePaymentsUtils.scala

```diff
@@ -57,4 +57,21 @@ object AccountFeePaymentsUtils {
       }
     }
   }
+
+  def getMainchainWithdrawalEpochDistributionCap(epochMaxHeight: Long, epochLength: Long): BigInteger = {
+    val baseReward = 12.5 * 1e8
+    val halvingInterval = 840000
+
+    var result = 0L
+    for (height <- epochMaxHeight - epochLength until epochMaxHeight) {
+      var reward = baseReward.longValue()
+      val halvings = height / halvingInterval
+      for (_ <- 1L to halvings) {
+        reward = reward >> 1
+      }
+      result = result + reward
+    }
+
+    BigInteger.valueOf(result).divide(BigInteger.valueOf(10))
```

Comment:
This percentage value is valid as of now, but in principle it might be modified in future releases. As a consequence we will be required to have this value dependent on a hardfork, therefore it would be better to access this value via an accessor method somehow or at least define a constant.